api/client/commands.go

```diff
@@ -222,6 +251,9 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	if *pull {
 		v.Set("pull", "1")
 	}
+
+	v.Set("f", *dockerfileName)
```

Comment:
Nonono this string is for the API parameter, not the CLI.  I 100% agree
with "-f" for the CLI.  This line represents the _API_.